docs-ref-conceptual/operationsmanager/index.md

```diff
@@ -48,3 +48,54 @@ Required action, applicable for Operations Manager 2019 UR1.
 **Example:**
 
 ![Initialize the CSRF token example](./Media/index/116854.png)
+
+## Call SCOM REST API from Powershell Script
```

Comment:
We don't use the acronym SCOM in our official documentation. Please rename to - Operations Manager REST API PowerShell example

Reply:
Hi @MGoedtel- Updated the name. Header would be displayed as H2 heading.